odxtools/comparam.py

```diff
@@ -1,60 +1,145 @@
 # SPDX-License-Identifier: MIT
+import warnings
 from dataclasses import dataclass
-from typing import TYPE_CHECKING, Any, Dict, List, Optional
+from typing import TYPE_CHECKING, Any, Dict, List, Optional, Union
 from xml.etree import ElementTree
 
-from .basecomparam import BaseComparam
-from .dataobjectproperty import DataObjectProperty
-from .exceptions import odxraise, odxrequire
+from .basecomparamspec import BaseComparamSpec
+from .comparamspec import ComparamSpec
+from .complexcomparamspec import ComplexComparamSpec, ComplexValue, create_complex_value_from_et
+from .exceptions import OdxWarning, odxassert, odxraise, odxrequire
 from .odxlink import OdxDocFragment, OdxLinkDatabase, OdxLinkId, OdxLinkRef
+from .utils import create_description_from_et
 
 if TYPE_CHECKING:
     from .diaglayer import DiagLayer
 
 
 @dataclass
-class Comparam(BaseComparam):
-    dop_ref: OdxLinkRef
-    physical_default_value: Optional[str]
+class Comparam:
```

Comment:
Another naming paradigm I have seen in FIBEX and AUTOSAR, is to use the "Instance" suffix, "Comparam" => define the com param schema, "ComparamInstance" => value of com param

Reply:
good suggestion. implemented: 968665c7430037d0e9d2ec772ef5524252f8df0f